app/helpers/comments_helper.rb

```diff
@@ -31,6 +31,16 @@ def should_be_hidden?(comment, root_comment)
     comment.hidden_by_commentable_user && comment != root_comment
   end
 
+  def like_button_text(comment)
+    if comment.public_reactions_count.zero?
+      "Like"
+    elsif comment.public_reactions_count == 1
+      "&nbsp;like"
+    else
+      "&nbsp;likes"
+    end
```

Comment:
**nitpick (non-blocking):** maybe we could use a case statement here if we feel it's easier to read:

```ruby
case comment.public_reactions_count
when 0
  "Like"
when 1
  "&nbsp;like"
else
  "&nbsp;likes"
end
```

Or if we want to import `pluralize` we can do something like:
```ruby
return "Like" if comment.public_reactions_count.zero?

"&nbsp;#{pluralize(comment.public_reactions_count, "like")}"

```

Reply:
I like either of these, but pluralize seems like a nice solution here :)

Reply:
I commented below before seeing this specific thread. Pluralize outputs the word _and_ the number so isn't on its own the thing, eh?

Reply:
@benhalpern you're right, that's my mistake! I gave the wrong example. We'd call pluralize on the string itself if we wanted to go this route:

```ruby
return "Like" if comment.public_reactions_count.zero?

"&nbsp;#{"like".pluralize(comment.public_reactions_count}"
```

Reply:
I added the case statement